README.md

```diff
@@ -29,6 +29,37 @@ For example:
 This will create a PrometheusRule based on the `5xx-rate.tmpl` template with a
 threshold of `0.001`.
 
+## Deployment Annotations
+
+Your Deployment must have annotations in the form of:
+
+`com.uswitch.heimdall/<prometheus-rule-name>: <threshold>`
+
+For example:
+
+`com.uswitch.heimdall/5xx-rate-deployment: "0.001"`
+
+This will create a PrometheusRule based on the `5xx-rate-deployment.tmpl` template with a
+threshold of `0.001`.
+
+Available annotations:
+
+`com.uswitch.heimdall/4xx-rate-deployment` - alerts if the 4XX rate goes above the given threshold for at least 1 minute
+`com.uswitch.heimdall/5xx-rate-deployment` - alerts if the 5XX rate goes above the given threshold for at least 1 minute
+`com.uswitch.heimdall/p95-deployment` - alerts if the P95 goes above the given ms for at least 5 minutes
+`com.uswitch.heimdall/p99-deployment` - alerts if the P95 goes above the given ms for at least 5 minutes
+`com.uswitch.heimdall/replicas-availability-deployment`- alerts if the given % of replicas are not running for 5 minutes
+
+## Running Heimdall locally
+
+Once the kubernetes context is set to a local cluster, skaffold + kustomize can help deploying the local Heimdall version
```

Comment:
link to what skaffold and kustomize are?

Reply:
Fair point, I am adding them as a link :+1: Update is to come shortly. Thank you